webpack/components/extensions/HostDetails/Tabs/ErratumExpansionContents.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  TreeView,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const ErratumExpansionContents = ({ erratum }) => {
+  const {
+    bugs, cves, packages,
+    module_streams: moduleStreams,
+  } = erratum;
+  const [activeItems, setActiveItems] = useState(null);
+  const options = [
+    {
+      name: __('Bugs'),
+      id: 'bugs',
+      children: bugs.map(bug => ({ name: bug.bug_id, id: bug.bug_id, ...bug })),
+    },
+    {
+      name: __('CVEs'),
+      id: 'cves',
+      children: cves.map(cve => ({ name: cve.cve_id, id: cve.cve_id, ...cve })),
+    },
+    {
+      name: __('Packages'),
+      id: 'packages',
+      // packages is just a list of strings
+      children: packages.map((packageName, idx) => ({ name: packageName, id: idx })),
+    },
+    {
+      name: __('Module streams'),
+      id: 'module_streams',
+      // module streams should already have name and id fields, so no need to map
+      children: moduleStreams,
```

Comment:
```suggestion
      children: moduleStreams.map(modStream => ({ name: `${modStream.name}:${modStream.stream}`, id: modStream.id})),
```
Include both name and stream when using module streams